doc/integrator/cluster.rst

```diff
@@ -0,0 +1,23 @@
+.. _integrator_cluster:
+
+Create a cluster
+================
+
```

Comment:
Avoid "we" in the doc => stay objective. I would write:
A cluster is a configuration where several servers are used to serve the application.